apps/web/public/static/locales/en/common.json

```diff
@@ -685,6 +685,7 @@
   "event_type_updated_successfully_description": "Your event type has been updated successfully.",
   "hide_event_type": "Hide event type",
   "edit_location": "Edit location",
+  "cant_find_the_right_video_app": "Can't find the right video app? Visit our",
```

Comment:
Maybe see how `broken_video_action` is implemented. 
"Can't find the right video app? Visit our App Store" should be a single sentence for translation. Breaking and concatenating strings after translation can have completely wrong meaning in some languages.